packages/package.go

```diff
@@ -319,15 +321,33 @@ func (p *Package) setRuntimeFields() error {
 
 	p.versionSemVer, err = semver.StrictNewVersion(p.Version)
 	if err != nil {
-		return errors.Wrap(err, "invalid package version")
+		return fmt.Errorf("invalid package version: %w", err)
 	}
 
 	if p.Conditions != nil && p.Conditions.Kibana != nil {
 		p.Conditions.Kibana.constraint, err = semver.NewConstraint(p.Conditions.Kibana.Version)
 		if err != nil {
-			return errors.Wrapf(err, "invalid Kibana versions range: %s", p.Conditions.Kibana.Version)
+			return fmt.Errorf("invalid Kibana versions range %s: %w", p.Conditions.Kibana.Version, err)
 		}
 	}
+
+	// Packages from proxy mode do not have "format_version" field
+	if p.FormatVersion == "" {
+		return nil
+	}
```

Comment:
But then `HasCompatibleSpec` won't work for proxied packages, right?

> And these proxied packages do not contain `format_version` in their struct/objects.

Why? Is it because we don't include `format_version` in search responses? Maybe we should add it now that it is meaningful for packages discoverability.

Reply:
> But then HasCompatibleSpec won't work for proxied packages, right?

I was thinking that as the query is sent to the remote (proxy) with the same parameters, the packages in the responses should be valid. 

> Why? Is it because we don't include format_version in search responses? Maybe we should add it now that it is meaningful for packages discoverability.

Exactly, currently `format_version` is not included in the responses.
It looks like including now this field in to the responses would be better. Should we added then in this PR ?

Reply:
> I was thinking that as the query is sent to the remote (proxy) with the same parameters, the packages in the responses should be valid.

Good point, that should be enough :+1: